db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -172,6 +172,17 @@ abstract class DbCommonsColumnMappers {
     MappedColumnType
       .base[SatoshisPerByte, Long](_.toLong, SatoshisPerByte.fromLong)
   }
+
+  implicit val addressTagMapper: BaseColumnType[AddressTagName] = {
```

Comment:
So anything that comes out of the database will be a `InternalAddress` type? Am I circling back to the `InternalDAO` vs `ExternalDAO` thing?

Reply:
Not sure what the value of having multiple DAOs is, could you elaborate?